src/UKHO.S100PermitService.API/Program.cs

```diff
@@ -141,15 +142,25 @@ private static void ConfigureServices(WebApplicationBuilder builder)
                 client.Timeout = TimeSpan.FromMinutes(holdingsServiceApiConfiguration.RequestTimeoutInMinutes);
             });
 
+            var userPermitServiceApiConfiguration = builder.Configuration.GetSection("UserPermitServiceApiConfiguration").Get<UserPermitServiceApiConfiguration>();
```

Comment:
Put the section names into const string so that if they are changed, all instances are updated

Reply:
Done